verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -506,11 +506,52 @@ paths:
 components:
   schemas:
     Source:
-      description: A complete source; a superset of "#/components/schemas/NewSource" with server-added fields
-      allOf:
-        - $ref: "#/components/schemas/NewSource"
-        - type: object
-          # TODO: Define Source object.
+      description: A source of line list case data.
+      type: object
+      properties:
+        name:
+          type: string
+        origin:
+          type: object
+          properties:
+            url:
+              type: string
+            license:
+              type: string
+        format:
+          type: string
+        automation:
+          type: object
+          properties:
+            parser:
+              type: object
+              properties:
+                awsLambdaArn:
+                  type: string
+                  pattern: '^arn\:aws\:lambda\:.+\:.+\:function\:.+'
+            regexParsing:
+              type: object
+              properties:
+                fields:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      name:
+                        type: string
+                      regex:
+                        type: string
+            schedule:
+              type: object
+              properties:
+                awsRuleArn:
+                  type: string
+                  pattern: '^arn\:aws\:events\:.+\:.+\:rule\/.+'
+                awsScheduleExpression:
+                  type: string
+                  pattern: '^((rate\(((1 (minute|hour|day))|([1-9]+ (minutes|hours|days)))\))|(cron\((.+\s){5}.+\)))'
```

Comment:
I missed this regex. Are we planning to leave these two in the Mongoose document as well?

Reply:
No personal preference -- I don't think it hurts but I guess it's not very DRY. WDYT?

Reply:
(Chatted separately -- leaving as-is in the document for now.)